services/anvil/l2_block_number.go

```diff
@@ -0,0 +1,30 @@
+package anvil
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ethereum-optimism/optimism/op-bindings/bindings"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// TODO connect this to superchain repo https://github.com/ethereum-optimism/mocktimism/issues/79
+func (a *AnvilService) L2BlockNumber(l1Client bind.ContractBackend, l2OutputOracleAddr common.Address, l1BlockNumber *big.Int) (*big.Int, error) {
```

Comment:
should the l2 output oracle addr be encapsulated in the service as that's non-changing?

Reply:
l2OutputOracle is on l1 it has a different addy on every l2